examples/grpc/helloworld_client/main.go

```diff
@@ -44,7 +44,7 @@ func main() {
 
 	// Set up a connection to the server with the OpenCensus
 	// stats handler to enable stats and tracing.
-	conn, err := grpc.Dial(address, grpc.WithStatsHandler(ocgrpc.NewClientStatsHandler()), grpc.WithInsecure())
+	conn, err := grpc.Dial(address, grpc.WithStatsHandler(new(ocgrpc.ClientHandler)), grpc.WithInsecure())
```

Comment:
This is not a common style in Go for inline code.

It is somewhat encouraged with var for zero values:

```
var hander = new(ocgrpc.ClientHandler)
```

Use `&ocgrpc.ClientHandler{}` instead.